django/core/serializers/json.py

```diff
@@ -62,7 +62,7 @@ def end_object(self, obj):
 
     def getvalue(self):
         # Grand-parent super
-        return super(PythonSerializer, self).getvalue()
+        return super().getvalue()
```

Comment:
Looking at your script, the intent seems correct, but the fact that this code was changed incorrectly proves that the implementation doesn't work (yet).